dev/build/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ghcr.io/ietf-tools/rpc-app-base:latest
+LABEL maintainer="IETF Tools Team <tools-discuss@ietf.org>"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN groupadd -g 1000 rpc && \
+    useradd -c "Purple People Eater" -u 1000 -g rpc -m -s /bin/false purple
```

Comment:
no way that won't come back as trouble later :P

Reply:
Should be ok, he wouldn't eat you 'cause you're so tough